tests/test_core.py

```diff
@@ -74,7 +74,45 @@ def setup_method(self):
         """
         self.dic_OP, self.dic_corresp_numres_index_dic_OP = init_dics.init_dic_OP(self.universe_woH,
                                                                                   self.dic_atname2genericname,
-                                                                                  self.dic_lipid)
+                                                                                  self.dic_lipid['resname'])
+
+
+    @pytest.mark.parametrize('index, Hs_coords', [
+            # atom C1 type CH3
+            (0,  (np.array([35.06161421, 47.69320272, 26.76728762]),
+                  np.array([33.93128850, 47.36328732, 25.43245201]),
+                  np.array([35.02249090, 46.08195972, 26.01188584]))),
+            # atom C5 type CH2
+            (4,  (np.array([31.62418725, 45.50474260, 27.61469385]),
+                  np.array([32.93474471, 44.52992542, 26.90727792]))),
+            # atom C13 type CH
+            (12, (np.array([26.61868981, 44.14296091, 25.55244500]),)),
+            # atom C24 type CHdoublebond
+            (23, (np.array([20.46454439, 38.99866187, 31.19224364]),)),
+    ])
+    def test_buildHs_on_1C(self, index, Hs_coords):
+        """
+        Test for buildHs_on_1C()
+        Generate 4 atoms to be tested, each with a different type
+        """
+        atom = self.universe_woH.atoms[index]
```

Comment:
To improve PEP 257 compliance:
```
"""Test for buildHs_on_1C().
 
Generate 4 atoms to be tested, each with a different type.
"""
```
Could you also fill `Parameters` and `Returns` fields?